.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,78 @@
+## Description
+
+Fixes #.
+
+Changes proposed in this pull request:
+
+## Type of change
+
+- [ ] Bug fix
```

Comment:
I just realized something, which is that I would recommend not using checkboxes in this way, because in other places around the github ui it registers checkboxes as tasks, which is not how you are using them:

![Screenshot 2023-07-27 at 2 51 56 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/912df078-2788-45d7-9f42-a1c862a4b416)

Perhaps for "Type of change" we should remove the checkboxes by default and have people just write the kind of change in directly instead of checking checkboxes.

Reply:
I would tend to agree with @dkaoster here. Let's make it section they enter as text and if desired you can add some guidance in the template on what they should enter

Reply:
Looks like sometime in the next few months the default behavior of HTML checkboxes might be changing: https://github.com/orgs/community/discussions/5197#discussioncomment-5966864

I don't have too strong an opinion here, just adding another data point.

Reply:
Oh interesting. In that case I may be inclined to keep this as is (and eventually it seems we'll have to converts the checks that *are* tasks to the new tasklist format)